### DISCUSSION TITLE
False positives

### DISCUSSION THREAD

**Discussion by malinb1** [globbestael/DedupEndNote#9] [General]
I'm getting false positives that seem very strange based on how the method is described. They are all conference papers and the authors are similar ("Assoc Informat, S" is also included as an author) but the titles are very different. How could this happen? 

These are treated as duplicates:

- Mushtaq, F., Tulu, B., Strong, D., Johnson, S., Trudel, J., Garber, L., & Assoc Informat, S. (2016). Explaining the Effect of Health Status on Patient Portal Use and Health System Utilization. [Amcis 2016 proceedings]. 22nd Americas Conference on Information Systems (AMCIS), San Diego, CA.
- Mushtaq, F., Tulu, B., Strong, D., Johnson, S., Trudel, J., Garber, L., & Assoc Informat, S. (2015, Aug 13-15). Analysis & Visualization of EHR Patient Portal Clickstream Data. [Amcis 2015 proceedings]. 21st Americas Conference on Information Systems (AMCIS), Fajardo, PR.
- Salina, E., Tulu, B., Johnson, S. A., Strong, D. M., Trudel, J., Garber, L., & Assoc Informat, S. (2016). Understanding the Link between Patient Portal Use and Health System Utilization. [Amcis 2016 proceedings]. 22nd Americas Conference on Information Systems (AMCIS), San Diego, CA.

These are also treated as duplicates:

- Ahmed, A., Nasralah, T., Wang, Y., Noteboom, C., Spohn, R., Plimpton, J. W., Parks, L., Shan, R. H., & Assoc Informat, S. (2017). Impact of E-Visits on Emergency Departments and Urgent Care Centers. [Amcis 2017 proceedings]. 23rd Americas Conference on Information Systems (AMCIS), Boston, MA.
- McCreless, T., Eid, W., & Assoc Informat, S. (2017). Impact of Patient Portal Use on Clinical Outcomes. [Amcis 2017 proceedings]. 23rd Americas Conference on Information Systems (AMCIS), Boston, MA.
- Samhan, B., Hauck, R. V., & Assoc Informat, S. (2018, Aug 16-18). Can Information Security Awareness Mitigate Resistance Towards Patients Healthcare Portals? Completed Research. [Amcis 2018 proceedings]. 24th Americas Conference on Information Systems (AMCIS) - Digital Disruption, New Orleans, LA.
- Venkatesan, S., Sharman, R., & Assoc Informat, S. (2016). Are Expert Patients Popular in Online Patient Portals? [Amcis 2016 proceedings]. 22nd Americas Conference on Information Systems (AMCIS), San Diego, CA.
- Venkatesan, S., Sharman, R., Das Smith, S., Prabhu, V., & Assoc Informat, S. (2018, Aug 16-18). Bi-modal Network Relationships in Online Health Networks Emergent Research Forum (ERF). [Amcis 2018 proceedings]. 24th Americas Conference on Information Systems (AMCIS) - Digital Disruption, New Orleans, LA.

**Comment by globbestael** [comment]
These are strange indeed. Could you send me the RIS records for these publications. DedupEndNote has only been tested on biomedical databases. From which database were they imported?

**Reply by malinb1** [reply]
These were exported from Web of Science. 

[tulu.txt](https://github.com/globbestael/DedupEndNote/files/12299428/tulu.txt)
[venkatesan.txt](https://github.com/globbestael/DedupEndNote/files/12299446/venkatesan.txt)

**Reply by renespijker** [reply]
@malinb1 just to be certain. The above files were imported into endnote and exported as an endnote ris file or was the web of science ris file directly uploaded onto dedupendnote?

**Reply by malinb1** [reply]
Imported to EndNote and exported from there as RIS again.

**Reply by renespijker** [reply]
thanks!

**Reply by renespijker** [reply]
@malinb1 when I load both the above files into endnote and export as ris see: [conferenece_wos_ris.txt](https://github.com/globbestael/DedupEndNote/files/12300023/conferenece_wos_ris.txt) and subsequently upload this file to dedupendnote no duplicates are found see:  ![Screenshot 2023-08-09 at 10 30 46](https://github.com/globbestael/DedupEndNote/assets/11136490/6ff37276-f1dc-4696-b782-131ac96c0557)

could you also send the final ris file you uploaded to the website

**Reply by malinb1** [reply]
That's strange. Here is the file I used. It's a mix of references from PubMed, WoS CC and PsycInfo but the conference references seem to come from WoS. 

[en-dedup.txt](https://github.com/globbestael/DedupEndNote/files/12300618/en-dedup.txt)

**Reply by renespijker** [reply]
@malinb1 when I run this file and use mark mode I'm not seeing the above references listed as duplicates
[en-dedup_mark.txt](https://github.com/globbestael/DedupEndNote/files/12300720/en-dedup_mark.txt)

you can use the mark mode and than load the ris file into endnote and display the label field and sort.
the conference abstracts listed above are not marked as duplicates in this run.
Could you rerun the file from your end using the mark option and see if you get the same result

**Reply by malinb1** [reply]
I reran the file I sent you and they still show up as duplicates in the mark mode. 
![image](https://github.com/globbestael/DedupEndNote/assets/124559384/43034fba-ba54-46e0-b2f1-6564b17aa297)

**Reply by malinb1** [reply]
"DONE: DedupEndNote has written 3562 records with 942 duplicates marked in the Label field."

**Reply by renespijker** [reply]
sorry for not responding but it's being investigated. thanks for providing us with all the material.

**Reply by globbestael** [reply]
Web of Science records were tested, but not on conference proceedings (we don't have a subscription to the conference proceedings). The title of the conference is put in the same EndNote field OP as is used for the original (non-English) title of PubMed records. I have deployed a new version which is w aware of this difference.
Thank for the files and your patience.
